conans/client/settings_preprocessor.py

```diff
@@ -9,6 +11,15 @@ def preprocess(settings):
     _check_cppstd(settings)
 
 
+def _make_cppstd_flag(compiler, compiler_version, cppstd):
+    settings = Settings.loads(get_default_settings_yml())
```

Comment:
We can't do this. The user might have their own settings file. We need to get here `cache.settings`  (see comment in the usage of this function)

Reply:
okay, let's use the cache

Reply:
seems like I don't have access to that here

Reply:
Yes, that's the reason why I was telling you (in the usage of this function) to keep using the old _fachade_ for the function.

Reply:
removed